cfgen/cfgen

```diff
@@ -1956,8 +1948,66 @@ def build_cluster(cluster_desc: Dict[str, Any]) -> Cluster:
                 proc_id = ConfProcess.build(conf, node_id, node, proc_t)
                 cluster.m0_clients[proc_id] = new_oid(ObjT.service)
 
+    # Create data structures required for aux pools
+    aux_pools: dict = {}
+    disk_ref_combinations = []
+    encl_ref_combination = []
+    tolerance: Failures
+    pool_nodes: list = []
+
     for pool in cluster_desc['pools']:
-        ConfPool.build(conf, root_id, pool)
+        pool_t = pool_type(pool)
+        if pool_t == PoolT.sns:
+            # Calculate and create aux pool disk combinations
```

Comment:
[Optional] Can this `if` branch be refactored into a separate function?

Reply:
Done!